{{cookiecutter.example_name}}/Code.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+# Copyright Insight Software Consortium
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import itk
+
+if len(sys.argv) != 3:
+    print("Usage: " + sys.argv[0] + "<InputFileName> "
+          "<OutputFileName>")
+    sys.exit(1)
+
+inputFileName = sys.argv[1]
+outputFileName = sys.argv[2]
+
+Dimension = 2
+PixelType = itk.UC
+ImageType = itk.Image[PixelType, Dimension]
+
+reader = itk.ImageFileReader[ImageType].New()
+reader.SetFileName(inputFileName)
+
+{{ cookiecutter.filter_instance }} = itk.{{ cookiecutter.class_name }}[ImageType, ImageType].New()
+{{ cookiecutter.filter_instance }}.SetInput(reader.GetOutput())
```

Comment:
Here we should use the snake case form of the class name.

Reply:
:thumbs: Is there a straightforward way to convert to snake case from the classical class name or else a somehow clever strategy would be needed in the cookiecutter `class_name` variable?

Reply:
This function can be used:

https://github.com/InsightSoftwareConsortium/ITK/blob/b85ae59b850972e4aea147c0cb0d3aabe1632faf/Wrapping/Generators/Python/itkHelpers.py#L19-L24

Reply:
👍 Done in 5409bf6.